### PR TITLE
Set the module name instead of relying on the default

### PR DESCRIPTION
**Pull request by timothymcmackin**
The smart contract tutorial uses the namespace/module  `Counter`. Conveniently, this is the default module name in the IDE, but we should still make sure that it is set correctly.

Screenshot 2024-03-11 at 10 53 18 AM

### CODE REVIEW

File: docs/tutorials/smart-contract/jsligo.mdx [hunk 1]
```diff
@@ -204,6 +204,12 @@ namespace Counter {
 
 Before you can deploy the contract to Tezos, you must compile it to Michelson, the base language of Tezos contracts.
 
+1. Set the compiler to compile the namespace in your code:
```
Comment (Laucans) [inline docs/tutorials/smart-contract/jsligo.mdx:207]: I would say : 
"to target the namespace to compile in your code

File: docs/tutorials/smart-contract/cameligo.mdx [hunk 2]
```diff
@@ -189,6 +189,12 @@ type returnValue = operation list * storage
 
 Before you can deploy the contract to Tezos, you must compile it to Michelson, the base language of Tezos contracts.
 
+1. Set the compiler to compile the namespace in your code:
```
Comment (Laucans) [inline docs/tutorials/smart-contract/cameligo.mdx:192]: Namespace is module in cameligo. 

Also I would say "to target the module to compile in your code"

File: docs/tutorials/smart-contract/jsligo.mdx (hunk 1 above)
Comment (timothymcmackin) [reply docs/tutorials/smart-contract/jsligo.mdx:207]: That's better, thank you. Updated/

**Review by beatalipska — DISMISSED** [review]
🚀

**Pull request merged**